src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -255,16 +257,17 @@ export const IframeRuntime: React.ForwardRefExoticComponent<IProps> = forwardRef
                   globalInteractiveState: null,
                   interactiveStateUrl: "",
                   collaboratorUrls: null,
-                  classInfoUrl: "",
+                  classInfoUrl: portalData?.portalJWT?.class_info_url ?? "",
                   interactive: {
                     id: 0,
                     name: ""
                   },
                   authInfo: {
                     provider: "",
```

Comment:
I dug through the LARA source and configuration. And I think this provider takes the form:
`https://learn.concord.org/`
You should be able to construct this by stripping off the path from from one of the URLs passed to the activity player during launch.